main.go

```diff
@@ -968,7 +968,7 @@ func LoadConfig() (*Config, error) {
 	}
 
 	if config.SSHConnConfig.Port == "" {
-		config.SSHConnConfig.Port = "ssh"
+		config.SSHConnConfig.Port = "22"
```

Comment:
This doesn't matter. Go looks up port service names and translates to the numerical value.

Reply:
updated https://github.com/jackc/tern/commit/d08e1b7273f60b94c8e27e83b8abb008789acde5